dataproc_jupyter_plugin/handlers.py

```diff
@@ -193,6 +194,11 @@ def full_path(name):
         "dagRunTask": airflow.DagRunTaskController,
         "dagRunTaskLogs": airflow.DagRunTaskLogsController,
         "clusterList": dataproc.ClusterListController,
+        "clusterListPage": cluster.ClusterListPageController,
```

Comment:
Drop this line entirely; there's no justification for having two different endpoints that make the exact same API call.